tests/legacy-cli/e2e/tests/generate/confg/type-karma.ts

```diff
@@ -1,6 +1,8 @@
 import { ng } from '../../../utils/process';
+import { useCIChrome } from '../../../utils/project';
 
 export default async function () {
   await ng('generate', 'config', 'karma');
+  await useCIChrome('test-project');
```

Comment:
This test must be new? This is required to update the karma config to use the headless chrome. See `project.ts`